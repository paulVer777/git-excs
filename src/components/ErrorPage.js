import React from 'react'
import {Link} from 'react-router-dom'

const ErrorPage = () => (
    <div>Error Page <Link to='/'>Home</Link></div>
)

export default ErrorPage