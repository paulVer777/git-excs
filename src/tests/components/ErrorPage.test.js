import React from 'react'
import {shallow} from 'enzyme'
import ErrorPage from '../../components/ErrorPage'

test('Should render error page correctly', () => {

    const wrapper = shallow(<ErrorPage />)
    expect(wrapper).toMatchSnapshot()
})