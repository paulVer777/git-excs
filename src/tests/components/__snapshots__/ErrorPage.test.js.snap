// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render error page correctly 1`] = `
<div>
  Error Page 
  <Link
    replace={false}
    to="/"
  >
    Home
  </Link>
</div>
`;
